wgpu/gui/base.py

```diff
@@ -330,8 +334,13 @@ def my_handler(event):
                 raise TypeError(f"Event types must be str, but got {type}")
 
         def decorator(_callback):
+            _callback.__dict__["__event_order"] = order
```

Comment:
I don't think its a good idea to store order on the callback object. E.g. the same function may be provided multiple times, for different event types, and with different orders. Also, callback may not allow setting attributes.

I think its better to instead do `self._event_handlers[type].append((order, _callback))`.